### ISSUE TITLE
Repeat Party Messages

### ISSUE THREAD

**Issue by SpySauce** [HabitRPG/habitica#1070]
I haven't used my party chat extensively up until now, but the messages are being repeated a LOT. Definitely 20+. Anyone else having this problem?

**Comment by StanLindsey** [comment]
This is a known bug, at the moment you can delete your own repeated messages until we develop a proper spam-validator. Thanks for the report though!

**Comment by lefnire** [comment]
I think "spam" isn't the right word really - it's model.push /
model.unshift duplicates Derby bug. this is one of the primary bits i'm
hoping 0.5 will fix

**Comment by AdrienLemaire** [comment]
I got a message repeated 100+ times. I had to delete them 3 times, because at some point, messages weren't deleted anymore (could still see many of them after refresh). And finally, all my party chat got deleted (I'm sure I clicked on the remove button next to my messages only).

At several other occasions, I thought I was posting messages, and after refreshing later, they disappeared (never got sent). That was a bit frustrating ^^'
